e2e-tests/playwright/support/server/default_config.ts

```diff
@@ -91,6 +91,7 @@ const defaultServerConfig: AdminConfig = {
         EnableIncomingWebhooks: true,
         EnableOutgoingWebhooks: true,
         EnableCommands: true,
+        OutgoingIntegrationRequestsTimeout: 30,
```

Comment:
Maybe it's better to name it as `OutgoingIntegrationRequestsTimeoutSeconds`

Reply:
3/5 that the godoc comment already make it clear that it's in seconds. I'm happy to also add it to the webapp files.